src/test/java/org/javarosa/core/storage/IndexedStorageUtilityTests.java

```diff
@@ -32,31 +32,39 @@ public abstract class IndexedStorageUtilityTests {
     Shoe[] eightSizesOfWomensNikes;
 
     Shoe[] fiveSizesOfMensVans;
+    Shoe[] fiveSortedSizesOfMensAdidas;
+
+    Shoe[] combinedCollection;
 
     protected abstract IStorageUtilityIndexed<Shoe> createStorageUtility();
 
     @Before
     public void setupStorageContainer() {
         storage = createStorageUtility();
 
-        nike = new Shoe("nike", "mens", "10");
+        nike = new Shoe("nike", "mens", 10);
 
         tenSizesOfMensNikes = new Shoe[10];
         for (int i = 0; i < 10; ++i) {
             tenSizesOfMensNikes[i] =
-                    new Shoe("nike", "mens", String.valueOf(i + 1));
+                    new Shoe("nike", "mens", i + 10);
         }
 
         eightSizesOfWomensNikes = new Shoe[8];
         for (int i = 0; i < 8; ++i) {
             eightSizesOfWomensNikes[i] =
-                    new Shoe("nike", "womens", String.valueOf(i + 1));
+                    new Shoe("nike", "womens", i + 20);
         }
 
         fiveSizesOfMensVans = new Shoe[5];
         for (int i = 0; i < 5; ++i) {
             fiveSizesOfMensVans[i] =
-                    new Shoe("vans", "mens", String.valueOf(i + 1));
+                    new Shoe("vans", "mens", i + 5);
+        }
+
+        fiveSortedSizesOfMensAdidas = new Shoe[5];
```

Comment:
as I see this is only used for matching an expected outcome of a test, so this must only be used in that particular test case and should not be written to db at all.